enterprise/models.py

```diff
@@ -41,6 +42,13 @@
 mark_safe_lazy = lazy(mark_safe, six.text_type)  # pylint: disable=invalid-name
 
 
+def mode_slugs():
+    """
+    Return course modes slugs
+    """
+    return json.dumps(["verified", "professional", "no-id-professional", "credit", "audit", "honor"])
```

Comment:
Please confirm if the order of the `mode_slugs` remains same on retrieval.
nit: single quotes.